great_expectations/checkpoint/checkpoint.py

```diff
@@ -457,6 +476,12 @@ def _add_to_store(self) -> None:
 
 @public_api
 class CheckpointResult(BaseModel):
+    """
+    The result of running a Checkpoint.
+
+    Contains the results of running each validation definition in the checkpoint.
```

Comment:
This definition feels a bit circular. I think the repetition of the word "result" is contributing to that feeling. I wonder if the second instance here could be rephrased. maybe something like

```suggestion
    The result of running a Checkpoint.

    Contains information about Expectation successes and failures from running each Validation Definition in the Checkpoint.
```